src/AccessLogEntry.cc

```diff
@@ -70,6 +70,14 @@ AccessLogEntry::syncNotes(HttpRequest *req)
         assert(notes == req->notes());
 }
 
+void
+AccessLogEntry::setReply(HttpReply *rep)
```

Comment:
If you want to properly fix the potential locking issues please consider making the reply member an HttpReplyPointer instead of adding this alternative way to set the member. I do not insist on this.

Reply:
Done at c5aaabc.